.github/workflows/deploy.yml

```
@@ -0,0 +1,23 @@
+on:
+    push:
+        branches:
+        - main
+jobs:
+    deploy:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Connect Tailscale
+            uses: tailscale/github-action@v2
+            with:
+              oauth-client-id: ${{ secrets.TS_OAUTH_CLIENT_ID }}
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 16 but found 14

(indentation)